i18n/fr/system76_keyboard_configurator.ftl

```diff
@@ -0,0 +1,77 @@
+-name = Configurateur de clavier
+
+app-about = À propos du {-name}
+app-title = System76 {-name}
+
+board-fake = {$model}, fausse
+
+button-cancel = Annuler
+button-configure = Configurer un clavier
+button-disable = Déactiver
+button-import = Importer
+button-test = Vérifier
+
+error-disable-key = Échec dans la désactivation de la clé.
```

Comment:
It seems that "touche" would be more appropriate than "clé' to refer to a key since "clé" is a litteral translation of "key" but isn't very fitting here.
This would be true for all other instances where "clé" is used.